index.html

```diff
@@ -5,10 +5,30 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>ITK - Simon</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Fredoka+One">
 </head>
 <body>
-    <!-- Good luck! -->
-
-    <script src="./index.js"></script>
+    <main class="main">
+        <section class="menuSection">
+          <h1 class="title">Simon Says!</h1>
+          <button class="startButton">START</button>
+            <!--  Added restart buttonn -->
+          <button class="startButton restartButton">RESTART</button>  
+        </section>
+    
+        <section class="gameSection">
+          <p class="levelLabel">Level 1 of 20</p>
+    
+          <div class="gameBox">
```

Comment:
Gabriel, I believe that we can use a <//article> semantic element because we could say that the 4 playing squares could be the focus of the website and article better reflects that.

Reply:
Gabriel here: Hello Carlos, yeah I can see why we could use an <//article> element but I'm not quite sure that an article is going be the best what about  a <//footer>

Reply:
Mmm, we need to search more about semantic elements. Then we can decide. :D